tests/pkg_openwsn_sock_udp/main.c

```diff
@@ -217,6 +217,25 @@ static void test_sock_udp_create__full(void)
     expect(_TEST_PORT_REMOTE == ep.port);
 }
 
+static void test_sock_udp_close__clean_queue(void)
+{
+    static const ipv6_addr_t src_addr = { .u8 = _TEST_ADDR_REMOTE };
+    static const ipv6_addr_t dst_addr = { .u8 = _TEST_ADDR_LOCAL };
+    static const sock_udp_ep_t local = { .family = AF_INET6,
+                                            .port = _TEST_PORT_LOCAL };
+    static const sock_udp_ep_t remote = { .addr = { .ipv6 = _TEST_ADDR_REMOTE },
+                                            .family = AF_INET6,
+                                            .port = _TEST_PORT_REMOTE };
+    static uint8_t test_data[] = "ABCD";
+
+    expect(0 == sock_udp_create(&_sock, &local, &remote, SOCK_FLAGS_REUSE_EP));
+    expect(_inject_packet(&src_addr, &dst_addr, _TEST_PORT_REMOTE,
+                            _TEST_PORT_LOCAL, test_data, sizeof(test_data),
+                            _TEST_NETIF));
+    sock_udp_close(&_sock);
+    expect(_check_net());
```

Comment:
In this particular case it checks that there are no packets in `openqueue`, as this is a controlled environment there can not be any packets others than the ones injected during the test. Although in theory there could be packets destined to another `sock` (but since the test is the only one injecting packets this does not happen). 

```
    /* queue must me empty */
    for (uint8_t i = 0; i < QUEUELENGTH; i++) {
        if (openqueue_vars.queue[i].owner != COMPONENT_NULL) {
            return false;
        }
    }
    return true;
}
```

Reply:
Should I add a comment:
```suggestion

    expect(0 == sock_udp_create(&_sock, &local, &remote, SOCK_FLAGS_REUSE_EP));
    /* inject a packet destined to _sock */
    expect(_inject_packet(&src_addr, &dst_addr, _TEST_PORT_REMOTE,
                            _TEST_PORT_LOCAL, test_data, sizeof(test_data),
                            _TEST_NETIF));
    /* _sock is closed before reading the packet, closing should drop it*/
    sock_udp_close(&_sock);
    expect(_check_net());
```

Reply:
ping :)

Reply:
Yes please. Comments always help :-)